### PR TITLE
Move fosshost and storj to IaaS, correct the CDN traffic of Arvan Cloud to 100GB / m, and add qingcloud DNS.

### PR DESCRIPTION
**Pull request by DreamOfIce**
Some Advence: merge 'Other Free Resources' and 'Miscellaneous'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1010,24 +1009,26 @@ Table of Contents
...
   * [Glauca](https://docs.glauca.digital/hexdns/) – Free DNS hosting for up to 3 domains and DNSSEC support
 
 **[⬆ back to top](#table-of-contents)**
 
 ## IaaS
 
-  * [filebase.com](https://filebase.com/) - S3 Compatible Object Storage Powered by Blockchain. 5 GB free storage for unlimited duration.
   * [backblaze.com](https://www.backblaze.com/b2/) — Backblaze B2 cloud storage. Free 10 GB (Amazon S3-like) object storage for unlimited time
-  * [scaleway.com](https://www.scaleway.com/en/object-storage/) — S3-Compatible Object Storage. Free 75 GB storage and external outgoing traffic
-  * [terraform.io](https://www.terraform.io/) — Terraform Cloud. Free remote state management and team collaboration for teams up to 5 users.
```
Comment (daroga0002) [inline README.md:1030]: Why you removing terraform.io ?

File: README.md [hunk 2]
```diff
@@ -1010,24 +1009,26 @@ Table of Contents
...
+  * [filebase.com](https://filebase.com/) - S3 Compatible Object Storage Powered by Blockchain. 5 GB free storage for unlimited duration.
+  * [fosshost.org](https://fosshost.org/) - Free open source hosting VPS, web, storage and mirror hosting.
+  * [scaleway](https://www.scaleway.com/en/object-storage/) — S3-Compatible Object Storage. Free 75 GB storage and external outgoing traffic.
```
[the remark this answers is not in the document]
Comment (DreamOfIce) [reply README.md:1040]: Ok!I change it

**Comment by ripienaar** [comment]
thanks

**Comment by ripienaar** [comment]
Sorry, I had to revert this - I didnt notice qingcloud did not have a english page, is there any english page I can look at?

**Comment by DreamOfIce** [comment]
Unfortunately, qingcloud doesn't seem to have an English page
I will delete it

**Pull request merged**